static/css/components/page-banner.less

```
@@ -7,13 +7,19 @@
 
 .page-banner {
   display: block;
-  color: @white;
+  color: @black;
```

```suggestion
  color: @white;
```
Black on a dark grey background is very hard to read.

---

Good catch, this was only supposed to be on `-mybooks`